recipes/zoofs/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "zoofs" %}
+{% set version = "0.1.25" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/zoofs-{{ version }}.tar.gz
+  sha256: 08a7af6e9315aee1f9e5a821b7aec64fe5a419ba273dfb6b869492704019f78d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - colorlog >=6.6.0
+    - numpy
+    - pandas >=1.3.0
+    - plotly >=5.6.0
+    - python >=3.7
+    - scipy >=1.4.1
+
+test:
+  imports:
+    - zoofs
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jaswinder9051998/zoofs
+  summary: zoofs is a Python library for performing feature selection using an variety of nature inspired wrapper algorithms..
+  license: Apache-2.0
+  license_file: LICENSE
+  description: |
+    zoofs is a Python library for performing feature selection using a variety of nature-inspired wrapper algorithms.
+    The algorithms range from swarm-intelligence to physics-based to Evolutionary.
+    It's easy to use , flexible and powerful tool to reduce your feature size.
+  doc_url: https://jaswinder9051998.github.io/zoofs/
+  dev_url: https://github.com/jaswinder9051998/zoofs/tree/development
+
+extra:
+  recipe-maintainers:
+    - Jaswinder
```

```suggestion
    - jaswinder9051998
```
This needs to be your GitHub username